src/python_pachyderm/mixin/pfs.py

```diff
@@ -75,6 +55,26 @@ def read(self, size=-1):
         return b"".join(buf)
 
 
+class PFSFile:
+    """
+    The contents of a file stored in PFS. You can treat these as
+    file-like objects, like so:
+
+    ```
+    source_file = client.get_file("montage/master", "/montage.png")
+    with open("montage.png", "wb") as dest_file:
+        shutil.copyfileobj(source_file, dest_file)
+    ```
+    """
+
+    def __init__(self, stream):
+        f = tarfile.open(fileobj=stream, mode='r|*')
```

Comment:
Maybe provide a brief explanation to why we are using `tarfile`?

Reply:
This is because GetFile can return an arbitrary directory tree. Tar is the format we decided on for serializing this structure.